packages/integration-tests/suites/replay/privacyInput/test.ts

```diff
@@ -0,0 +1,108 @@
+import { expect } from '@playwright/test';
+import { IncrementalSource } from '@sentry-internal/rrweb';
+import type { incrementalData, inputData } from '@sentry-internal/rrweb/typings/types';
+
+import { sentryTest } from '../../../utils/fixtures';
+import {
+  getIncrementalRecordingSnapshots,
+  shouldSkipReplayTest,
+  waitForReplayRequest,
+} from '../../../utils/replayHelpers';
+
+function isInputMutation(snap: incrementalData): snap is inputData {
+  return snap.source == IncrementalSource.Input;
+}
+
+sentryTest(
+  'should mask input initial value and its changes',
+  async ({ browserName, forceFlushReplay, getLocalTestPath, page }) => {
+    // TODO(replay): This is flakey on firefox where we do not always get the latest mutation.
+    if (shouldSkipReplayTest() || browserName === 'firefox') {
```

Comment:
How do yall feel about this lol, I was not able to get this to work reliably on firefox  @Lms24 @mydea

Reply:
I think @Lms24 did the same for some other test, IMHO it is fine.

Reply:
Yup, let's skip this on FF